build/ProjectReunion-CI.yml

```diff
@@ -38,11 +38,38 @@ jobs:
   - template: AzurePipelinesTemplates\ProjectReunion-BuildDevProject-Steps.yml
   - template: AzurePipelinesTemplates\ProjectReunion-PublishProjectOutput-Steps.yml
 
-# MRTCore build pipeline
-- template: ..\dev\MRTCore\azure-pipelines.yml
-  parameters:
-    buildJobName: 'BuildMRTCore'
-    
+- job: BuildMRT
+  pool:
+    vmImage: 'windows-latest'
+  strategy:
+    maxParallel: 10
+    matrix:
+      Release_x86:
+        buildPlatform: 'x86'
+        buildConfiguration: 'Release'
+      Release_x64:
+        buildPlatform: 'x64'
+        buildConfiguration: 'Release'
+      Release_Arm64:
+        buildPlatform: 'ARM64'
+        buildConfiguration: 'Release'
+  steps:
+  - task: NuGetAuthenticate@0
+    inputs:
+      nuGetServiceConnections: 'WindowsES'
```

Comment:
why is this not needed in ProjectReunion-BuildFoundation.yml?

Reply:
It's because calling NuGetAuthenticate on the internal pipeline will cause nuget restore to fail. I don't understand why.

Reply:
probably worth a comment

---
In reply to: [581568235](https://github.com/microsoft/ProjectReunion/pull/473#discussion_r581568235) [](ancestors = 581568235)

Reply:
Will add a comment later